dbt_subprojects/dex/models/_projects/sushiswap/apechain/_schema.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+models:
+  - name: sushiswap_agg_apechain_trades
+    meta:
+      blockchain: apechain
+      sector: dex
+      project: sushiswap
+      contributors: olastenberg
+    config:
+      tags: [ 'apechain','sushiswap','trades','dex','aggregator' ]
+    description: Sushiswap Route Aggregator contract on apechain - formatted for dex.trades
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+            combination_of_columns:
+            - tx_hash
+            - method
+            - trace_address
```

Comment:
even if it doesn't matter, for readability, can you match the list here to model config?
`['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'method', 'trace_address'],`